lms/static/scripts/frontend_apps/components/dashboard/DashboardActivityFilters.tsx

```diff
@@ -51,8 +51,10 @@ export type ActivityFilterItem<T extends Course | Assignment> = {
   onClear: () => void;
 };
 
+export type SegmentsType = 'groups' | 'sections' | 'unknown';
```

Comment:
I think `"none"` would be clearer than `"unknown"` here, since the meaning is not that we don't know what subdivision of students is being used, but that _no_ subdivision is used. `null` could also be used.

Reply:
Yep, good idea.